mico-admin/src/app/dialogs/service-picker/service-picker.component.html

```diff
@@ -41,8 +46,24 @@ <h2 mat-dialog-title>Pick a Service</h2>
 
 
         <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
-        <tr mat-row *matRowDef="let row; columns: displayedColumns;" (click)="selection.toggle(row)">
-        </tr>
+        <div *ngIf="choiceModel==1">
```

Comment:
```suggestion
        <ng-container *ngIf="choiceModel==1">
```

use ng-container for ngIf because it doesn't show up in final dom that way.